scripts/README

```diff
@@ -0,0 +1,21 @@
+All scripts should be run from the root of the repo, like:
+
+```
+$ pwd
+xxxxx/photoview
+$ ./scripts/up_database.sh
+$ ./scripts/compose.sh run -it api /bin/bash
+...
+$ ./scripts/test_database.sh
+$ ./scripts/down_database.sh
+```
+
+- For developing:
+  - compose.sh: run `docker compose -f dev-compose.sh`. It accepts parameters after `docker compose`.
+  - up_database.sh: launch mysql and postgres in the background.
+  - down_database.sh: stop databases and clean files under `./dev`.
+- For building:
+  - install_build_dependencies.sh: install API dependencies for building.
+  - set_compiler_env.sh: setup the compiler variables for building API.
+- For testing:
+  - test_database.sh: test API against database.
```

Comment:
Is it possible to run other API tests? Like for scanner? Those, not directly testing the DB, but focused on other areas

Reply:
I'm thinking about making it more general. Currently `-database` is special, as it only works with `-p 1` parameters.

`test_database.sh` currently accepts other parameters to run tests, like `./scripts/test_database.sh mysql -- -coverprofile=coverage.txt -covermode=atomic`. It runs with MySQL database and generates the coverage report.